utils.py

```diff
@@ -51,9 +52,9 @@ def prune_empty(row):
     return new_row
 
 
-def chunks(l, n):
-    n = max(1, n)
-    return (l[i:i + n] for i in range(0, len(l), n))
+def chunks(mylist, n):
```

Comment:
Would prefer this to be my_list to be consistent with the naming most other places

Reply:
Fixed in latest commit